man/subscription-manager.8

```diff
@@ -1027,10 +978,6 @@ Overall Status: Insufficient
 .SS DEPRECATED COMMANDS
 As the structures of subscription configuration have changed, some of the original management commands have become obsolete. These commands have been replaced with updated commands.
 
-.TP
-.B subscribe
```

Comment:
Since you are removing this one, also remove its mention on line 141/138

> `Following commands were deprecated: addons, role, service-level, subscribe, unsubscribe, usage, and activate`

Reply:
Fixed